calcite/demo/main.cpp

```diff
@@ -41,7 +41,7 @@ main(int argc, char* argv[])
   // (https://links.esri.com/arcgis-api-keys).
 
   const QString apiKey =
-    QString("");
+      QString("");
```

Comment:
is this spacing correct?

Reply:
maybe no. I guess it was the default indentation of Qt Creator. I guess it's better to remove the new line.

Reply:
Oh, I forgot it's calcite. I tried to fix it in first time, but it's too complicated and I saw there is an issue for calcite. I canceled my changes.